decode/render.ts

```diff
@@ -218,10 +218,11 @@ function setAttributes(node: HTMLElement, attributes: object): void {
     for (let attribute in attributes) {
         if (attributes[attribute] !== undefined) {
             try {
+                let v = unmask(attributes[attribute]);
                 if (attribute.indexOf("xlink:") === 0) {
-                    node.setAttributeNS("http://www.w3.org/1999/xlink", attribute, attributes[attribute]);
+                    node.setAttributeNS("http://www.w3.org/1999/xlink", attribute, v);
                 } else {
-                    node.setAttribute(attribute, attributes[attribute]);
+                    node.setAttribute(attribute, v);
```

Comment:
>attribute [](start = 38, length = 9)

Will masking attribute values affect playback for some attributes?

Reply:
No, shouldn't have an impact of the playback. Before rendering we are attempting to unmask the text if it may already have been masked during the instrumentation.